authentication/app/routes/employee.js

```diff
@@ -0,0 +1,59 @@
+Employee = require('../controller/employeeController');
+Project = require('../controller/projectController');
+
+const errorHandler = require('../errorHandler'); 
+module.exports = function(app) {
+  app.post("/api/employees", function(req, res) {
```

Comment:
I wonder if this an API or a mess. In case of an error, response is unknown. Quiet a guessing game.